tests/test_migration.py

```diff
@@ -80,7 +80,7 @@ def test_db_migration(self, init_env: None) -> None:
                 convert_table_md(env.engine, substitution_fn=self.__substitute_md)
 
             # make sure we run the env db setup
-            Env.clear()
+            Env._init_env()
```

Comment:
Does this need to be called with `reinit_db=True`?

Reply:
It fails with reinit_db=True.